docs/explanations/Azure/Focal_To_Jammy_Upgrade.md

```diff
@@ -0,0 +1,35 @@
+# Upgrading from Focal to Jammy on Azure
+
+## General Advice
+When deciding how to upgrade the main consideration is whether your system can be setup/deployed with automation or requires a deal of manual configuration. 
+
+Running [`do-release-upgrade`](https://manpages.ubuntu.com/manpages/focal/man8/do-release-upgrade.8.html) requires some manual intervention ([see Manual upgrade steps](#manual-upgrade-steps)), which makes it a good option for systems which require manual configuration and cannot be easily created or destroyed. 
+
+For system deployments which are fully automated it is recommended to redeploy with new Jammy instances instead of upgrading from Focal.
+
+## Manual upgrade steps
+If you are upgrading from Focal to Jammy you can expect to run into the following prompts requesting manual input.
+
+### Additional SSH daemon
+![Additional SSH daemon](/docs/explanations/Azure/Focal_To_Jammy_Upgrade_Images/0_additional_ssh_daemon.png)
```

Comment:
let's add short written descriptions under the heading, and alt-text for the images describing them